charts/pega/config/deploy/context.xml.tmpl

```diff
@@ -28,4 +28,7 @@
   <Environment name="prconfig/database/databases/CustomerData/defaultSchema" value="{{ .Env.DATA_SCHEMA }}" type="java.lang.String" />
   {{ end }}
   <Environment name="prconfig/database/drivers" value="com.microsoft.sqlserver.jdbc.SQLServerDriver;oracle.jdbc.OracleDriver;org.postgresql.Driver;com.ibm.db2.jcc.DB2Driver" type="java.lang.String" />
+  {{ if .Env.WEB_REQUESTOR_PASSIVATION_TIMEOUT }}
+  <Environment name="prconfig/timeout/browser" value="{{ .Env.WEB_REQUESTOR_PASSIVATION_TIMEOUT }}" type="java.lang.String" />
```

Comment:
Remove WEB_ and just call the variable REQUESTOR_PASSIVATION_TIMEOUT to match the Helm parameter?

Reply:
Changed the name to REQUESTOR_PASSIVATION_TIMEOUT